requirements/worker_production.txt

```diff
@@ -0,0 +1,8 @@
+matplotlib==2.2.3
```

Comment:
Cannot we do like `-r worker.txt`instead of replicating the same requirements here?